scripts/minikube-k8sibmc-setup-env.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n.---------------------------------------\n${1}\n'---------------------------------------\n"
+  set -x
+}
+
+# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
+export HOME=`pwd`/home
+export GOPATH="`pwd`/home/go"
+export GOBIN="$GOPATH/bin"
+mkdir -p $GOBIN
+# This si one of the variables injected by ci-firewall. Its purpose is to allow scripts to handle uniqueness as needed
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+
+# Add GOBIN which is the bin dir we created earlier to PATH so any binaries there are automatically available in PATH
+export PATH=$PATH:$GOBIN
+
+# Prep for integration/e2e
+shout "Building odo binaries"
+make bin
+
+# copy built odo to GOBIN
+cp -avrf ./odo $GOBIN/
+
+setup_kubeconfig() {
+    export KUBECONFIG=$HOME/.kube/config
+    if [[ ! -f $KUBECONFIG ]]; then
+        echo "Could not find kubeconfig file"
+        exit 1
+    fi
+    if [[ ! -z $KUBECONFIG ]]; then
+        # Copy kubeconfig to current directory, to avoid clashes with other test runs
+        # Read and Write permission to current kubeconfig file
+        cp $KUBECONFIG "`pwd`/config"
+        chmod 640 "`pwd`/config"
+        export KUBECONFIG="`pwd`/config"
+    fi
+}
+
+case ${1} in
+    k8s)
+        setup_kubeconfig
+        ;;
+    minikube)
+        mkStatus=$(minikube status)
+        shout "| Checking if Minikube needs to be started..."
+        if [[ "$mkStatus" == *"host: Running"* ]] && [[ "$mkStatus" == *"kubelet: Running"* ]]; then 
+            if [[ "$mkStatus" == *"kubeconfig: Misconfigured"* ]]; then
+                minikube update-context
+            fi
+            setup_kubeconfig
+            kubectl config use-context minikube
+        else
+            minikube delete
+            shout "| Start minikube"
+            minikube start --vm-driver=docker --container-runtime=docker
+            setup_kubeconfig
+        fi
+        
+        minikube version
+        ;;
+    *)
+        echo "<<< Need parameter set to minikube or minishift >>>"
+        exit 1
+        ;;
+esac
+
+### Applies to both K8S and minikube
+# Setup to find nessasary data from cluster setup
+## Constants
+SETUP_OPERATORS="./scripts/configure-cluster/common/setup-operators.sh"
+
+# The OLM Version
+export OLM_VERSION="v0.17.0"
```

Comment:
Why? 0.18.3 works just fine. And why not use `latest` so that we don't need to keep updating with every release? To use `latest`, you'll have to use a third-party service like https://gitreleases.dev/, as GH doesn't provide a straightforward way to download assets of the latest release.

Reply:
setting it to 0.18.3 for now